js/background/background.js

```diff
@@ -437,137 +424,138 @@ class SteamCommunity extends Api {
      * Inventory functions, must be signed in to function correctly
      */
     static async coupons() { // context#3
-        let self = SteamCommunity;
-
-        let coupons = CacheStorage.get("inventory_3", 3600);
-        if (!coupons) {
-            coupons = {};
-            let data = await self.getInventory(3);
-
-            for (let description of data.descriptions) {
-                if (!description.type || description.type !== "Coupon") { continue; }
-                if (!description.actions) { continue; }
-
-                let coupon = {
-                    "image_url": description.icon_url,
-                    "title": description.name,
-                    "discount": description.name.match(/([1-9][0-9])%/)[1],
-                    "id": description.classid + '_' + description.instanceid
-                };
-                description.descriptions.forEach((desc, i) => {
-                    let value = desc.value;
-                    if (value.startsWith("Can't be applied with other discounts.")) {
-                        Object.assign(coupon, {
-                            "discount_note": value,
-                            "discount_note_id": i,
-                            "discount_doesnt_stack": true,
-                        });
-                    } else if (value.startsWith("(Valid")) {
-                        Object.assign(coupon, {
-                            "valid_id": i,
-                            "valid": value,
-                        });
-                    }
-                });
-                
-                for (let action of description.actions) {
-                    let match = action.link.match(/[1-9][0-9]*(?:,[1-9][0-9]*)*/);
-                    if (!match) {
-                        console.warn("Couldn't find packageid(s) for link %s", action.link);
-                        continue;
-                    }
+        let coupons = {};
+        let data = await SteamCommunity.getInventory(3);
+
+        for (let description of data.descriptions) {
+            if (!description.type || description.type !== "Coupon") { continue; }
+            if (!description.actions) { continue; }
+
+            let coupon = {
+                "image_url": description.icon_url,
+                "title": description.name,
+                "discount": description.name.match(/([1-9][0-9])%/)[1],
```

Comment:
so, discounts can be at least 10%? No 0-9% discounts allowed?

Reply:
It seems so.
I did not write this code though